src/main/java/core/basesyntax/operation/SupplyOperation.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.operation;
```

Comment:
The `operation` package should be under the `strategy` package to adhere to the checklist requirement of applying the Strategy pattern and organizing code into appropriate packages.